.gitignore

```diff
@@ -3,5 +3,7 @@
 .idea/
 sites/*/sqlconf.php
 sites/*/profile_pictures/*
+sites/*
+!sites/default
```

Comment:
I think this will do @Patil2099 . Only one thing removing this `sites/*/profile_pictures/*` cause `sites/*` will take care of that and also `sites/*/sqlconf.php` . Now here is the thing we want the stuffs under **default** to show up if modified , altered by git except for its `sqlconf.php` cause absolutely this has caused trouble :). The rest can be discarted.

Once more welcome and Nice having you onboard!!